tests/__fixtures__/Rules/only_given_step_in_background/fixture.js

```diff
@@ -0,0 +1,170 @@
+const generator = require("../../../helpers/problemGenerator");
+const OnlyGivenStepInBackground = require("../../../../lib/rules/only_given_step_in_background");
+
+function generateProblem(location) {
+    return generator(
+        OnlyGivenStepInBackground,
+        location,
+        OnlyGivenStepInBackground.meta.message
+    );
+}
+
+function getValidTestData() {
+    return [
+        [
+            "with Rule: Background",
+            `Feature: a feature file
+  Rule: a rule
+    Background: a background
+      Given a step`,
+            [],
+        ],
+        [
+            "with Rule: Background - Scenario",
+            `Feature: a feature file
+  Rule: a rule
+    Background: a background
+      Given a step
+      Given a step
```

Comment:
I suppose with just this rule, this passes.
And it will fail the `no_repetitive_step_keyword` rule.

Reply:
yes, the test data here are only for the rule being tested